network/src/commonMain/kotlin/com/wire/kalium/network/api/pagination/PaginationRequest.kt

```diff
@@ -0,0 +1,12 @@
+package com.wire.kalium.network.api.pagination
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class PaginationRequest(
+    @SerialName("paging_state")
+    val pagingState: String?,
+    @SerialName("size")
+    val size: Int? = null // Set in case you want specific number of pages, otherwise, the backend will return default per endpoint
```

Comment:
Perhaps worth to mention what the default size is ? I even wonder if we want to relay on backend default size?

Reply:
It depends, from endpoint to endpoint, for example connection requests default size is 100, for conversations 1000 😅 
So the idea is you can handle on the client/endpoint if you want something specific

Reply:
Small details but what would be the case to relay on backend default values wouldnt it make more sense to have it set on the clients?